File: bubblesort.c
#include <stdio.h>
int main()
{
int array[4]={6,4,5,2};
int count=4,temp;
while(count){
for(int i=0;i<4;i++){
if(array[i]>array[i+1]){
temp=array[i];
array[i]=array[i+1];
array[i+1]=temp;
}
}
count=count-1;
}
for(int i=0;i<4;i++){
printf("%d",array[i]);
}
return 0;
}

File: bubblesort.py
# vs code
array=list(map(int, input("elements of array:-").strip().split()))
print(array)
length=len(array)
print(length)
count=length
while(count): 
    for i in range(0,count-1):
        if array[i] > array[i+1]:
            temp=array[i]
            array[i]=array[i+1]
            array[i+1]=temp
        for i in range(length):
            print(array[i],end=" ")
        print(end=" ")
           
    print(" ")        
    count=count-1  
   